api/service/syncing/downloader/server.go

```diff
@@ -24,7 +25,14 @@ type Server struct {
 
 // Query returns the feature at the given point.
 func (s *Server) Query(ctx context.Context, request *pb.DownloaderRequest) (*pb.DownloaderResponse, error) {
-	response, err := s.downloadInterface.CalculateResponse(request)
+	var pinfo string
```

Comment:
better add a comment on why you are doing this.

Reply:
this is for debugging purpose. it's kind of overlap with selfip and selfport in your comment below since it will not contain the correct port number in local test

Reply:
comment added